server/app/models/Account.java

```diff
@@ -160,4 +161,11 @@ public String getApplicantName() {
         .map(u -> u.getApplicantData().getApplicantName().orElse("<Unnamed User>"))
         .orElse("<Unnamed User>");
   }
+
+  public Optional<LocalDate> getApplicantDateOfBirth() {
+    return this.getApplicants().stream()
+        .max(Comparator.comparing(Applicant::getWhenCreated))
+        .map(u -> u.getApplicantData().getDateOfBirth())
```

Comment:
nit: use method reference

```suggestion
        .map(Applicant::getApplicantData)
        .map(ApplicantData::getDateOfBirth)
```